### DISCUSSION TITLE
Need a custom domain

### DISCUSSION THREAD

**Discussion by sauravbhattacharya001** [knowledgefutures/pubpub#3080] [Support]
Hi, we have recently created our peer-reviewed open access journal International Journal of Global Innovations and Solutions (IJGIS) on PubPub - https://ijgis.pubpub.org.

I am looking to get a custom domain so I can add it the pages to be indexed in Google search console. Can you please tell me the process for that?

**Comment by eastofesten** [comment]
Hello! Our guide for adding a custom domain or subdomain for your community can be found [here](https://help.pubpub.org/pub/jzwe5q1g/release/12).
